src/svc-hooks/create-init.ts

```diff
@@ -22,8 +21,9 @@ export function handler(argv: any) {
       eventType: "build.complete",
       url: null,
       eventSpecificArgs: {
-        definitionName: null,
-        buildStatus: null,
+        pipelineName: "'' | <pipeline-name>",
```

Comment:
Can we leave these as null but add the options as comments?  That way the YAML still parses and tokenized correctly by editors...